chunky/src/java/se/llbit/chunky/resources/HDRTexture.java

```diff
@@ -102,7 +102,7 @@ public HDRTexture(File file) {
         }
       }
     } catch (IOException e) {
-      Log.error("Error loading HRD image: " + e.getMessage());
+      Log.error("Error loading HRD image:", e);
```

Comment:
Should this not be `HDR`? I assume this was a typo as the file name is the correct order.

Reply:
Fixed.